File: Python/tests/test_pst_gainloss_Q.py
import pytest

from hbayesdm.models import pst_gainloss_Q


def test_pst_gainloss_Q():
    _ = pst_gainloss_Q(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_pst_gainloss_Q.R
context("Test pst_gainloss_Q")
library(hBayesDM)

test_that("Test pst_gainloss_Q", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(pst_gainloss_Q(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
